docs/Maintainers.md

```diff
@@ -0,0 +1,50 @@
+Maintainers
+===========
+
+This document describes the Maintainers Committee of Open Enclave. By our
+liberal contribution policy outlined in our
+[governance model](GovernanceModel.md), maintainers are committers that are
+trusted to grant new committer rights, and grant new membership into the
+Maintainers Committee.
+
+When making decisions, the Maintainers Committee uses a "consensus seeking"
+process. This means that most decisions should be reached by consensus, but when
+that fails, the committee calls for a vote where the majority wins. This is to
+prevent obstructionism by removing the possibility of a one person veto.
+
+Maintainers Committee Members
+-----------------------------
+
+PENDING
+
+Responsibilities
+----------------
+
+The primary responsibility of the committee is to grant new committer rights
+(that is, write access to the main Open Enclave SDK repository or related
+repositories), and to grant new membership into the committee. Conversely, the
+committee must also remove committer rights and membership from those found to
+be violating the project's Code of Conduct or otherwise negatively affecting the
+project's community health.
+
+This committee is not intended to make every technical decision, as those should
+generally be made by agreement among committers as PRs are reviewed and merged.
+Where disagreements take place and need further resolution, those can be brought
+up with the Maintainers Committee as part of its responsibility to maintain the
```

Comment:
There's a weird split here suggesting that the committee is not responsible for technical steering but is still required to be a tie breaker on technical issues. Is there a separate steering committee? If so, what's the relationship between the two committees? If we say that there is no steering committee and there is no technical direction from the maintainers, then what is expected of committers? Given that these docs define committers and maintainers in relation to each other, I don't think we can get away with not also detailing the responsibilities and expectations of committers.

Reply:
No, I was not defining a separate steering committee. Our current governance model is the liberal contribution policy, that is, technical decisions are made by collaboration among the most active committers. It's really only if they reach a deadlock that they might request a more formal intervention by the Maintainers Committee (and it happens at that point because it's become a community health issue).

Reply:
You're using the phrase "liberal contribution policy" as if it is a widely understood phrase with a generally accepted set of conventions. Is it? Node.js has a set of principles that they follow, for example, but they quantify what it means to do this. For example, are we going to let every PR automatically merge by default if not reviewed? There needs to be some expectation that maintainers behave in a consistent fashion that is not going to be surprising to contributors submitting PRs in good faith (whether it's here or expanding the governance doc is an implementation detail).

Reply:
I'm sorry, I meant to be using it is a shared understanding of how we've already been developing: the active contributors in the project work together to merge PRs and so lead the technical direction of the project, without any formal voting or meeting to do so (that would be a "meritocracy."

This is a fairly common open source practice (to the point that it is detailed in [opensource.guide](https://opensource.guide/leadership-and-governance/#what-are-some-of-the-common-governance-structures-for-open-source-projects), which is what we gleaned our original docs from). Node.js has done an excellent job explaining how it works, and those principles are what we wrote as our intention to follow in our [governance docs](https://github.com/openenclave/openenclave/blob/master/docs/GovernanceModel.md). We explicitly chose _not_ to develop a formal voting process for (most) technical decisions.